app/helpers/admin/breweries_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Admin::BreweriesHelper
```

Comment:
__[rubocop]__
* Missing top-level module documentation comment.
* Use nested module/class definitions instead of compact style.